Library/Homebrew/test/formulary_spec.rb

```diff
@@ -136,20 +136,26 @@ class Wrong#{described_class.class_s(formula_name)} < Formula
     end
 
     context "with installed Formula" do
-      let(:formula) { described_class.factory(formula_path) }
-      let(:installer) { FormulaInstaller.new(formula) }
+      before do
+        allow(Formulary).to receive(:loader_for).and_call_original
+        stub_formula_loader formula("patchelf") { url "patchelf-1.0" }
+        allow(Formula["patchelf"]).to receive(:installed?).and_return(true)
+      end
+
+      let(:installed_formula) { described_class.factory(formula_path) }
+      let(:installer) { FormulaInstaller.new(installed_formula) }
 
       it "returns a Formula when given a rack" do
         installer.install
 
-        f = described_class.from_rack(formula.rack)
+        f = described_class.from_rack(installed_formula.rack)
```

Comment:
The name `formula` conflicted with the module method `formula` in `Test::Helper::Formula`.